schema.json

```diff
@@ -108,7 +108,9 @@
             "type": "string",
             "enum": [
               "open_question",
-              "closed_question"
+              "closed_question",
+              "form_filling",
+              "upload_document"
```

Comment:
Personally I would leave the upload_document as we would upload a link to a document but for now I would keep it so we can open this discussion still later as it requires further research